crnlib/lzma_LzmaEnc.cpp

```diff
@@ -1972,6 +1972,7 @@ SRes LzmaEnc_CodeOneMemBlock(CLzmaEncHandle pp, Bool reInit,
   p->rc.outStream = &outStream.funcTable;
 
   res = LzmaEnc_CodeOneBlock(p, True, desiredPackSize, *unpackSize);
+  p->rc.outStream = nullptr;

```

Comment:
Why?

Reply:
I don't reproduce anymore that dangling-pointer warning, maybe it was a side effect of something else now fixed by another commit.